packages/cdktf/lib/terraform-stack.ts

```diff
@@ -72,6 +73,14 @@ export class TerraformStack extends Construct {
     curr[lastKey] = value;
   }
 
+  public useConstructIdsAsNames() {
```

Comment:
Couldn't this be an option for the constructor?

Reply:
It could be. I was going for more of a this is something non-default to opt in to, but maybe that's just my Java background showing through.

Or maybe this is actually more of a long term different mode that people would continue to use than just as a transition.